src/cs50.c

```diff
@@ -340,9 +361,17 @@ string get_string(struct prompt *p)
     int c;
 
     // prompt user
-    if (p->prompt != NULL)
```

Comment:
Where would be the best place to do this since every `get` function is variadic?